.github/workflows/firefox-ios-import-strings.yml

```
@@ -49,8 +49,8 @@ jobs:
       run: sh ./bootstrap.sh --importLocales
     - name: Update new strings
       run: |-
-        git diff || (git add Shared/*/*.lproj/* Shared/*.lproj/* WidgetKit/*.lproj/* Client/*/*.lproj/* Client/*.lproj/*)
-        git restore Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved
+        git diff || (git add firefox-ios/Shared/*/*.lproj/* firefox-ios/Shared/*.lproj/* firefox-ios/WidgetKit/*.lproj/* firefox-ios/Client/*/*.lproj/* firefox-ios/Client/*.lproj/*)
+        git restore firefox-ios/Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved
     - name: Create Pull Request
       uses: peter-evans/create-pull-request@v4
```

```suggestion
      uses: peter-evans/create-pull-request@v5
```

There is at least a `Buffer()` deprecation fixed in v5.

Also checkout is available in v4 (currently v3), setup-python in v5 (currently v3)